packages/react-components/src/localization/locales/en-US/strings.json

```diff
@@ -183,5 +183,10 @@
     "participantStateConnecting": "Calling...",
     "participantStateRinging": "Calling...",
     "participantStateHold": "On hold"
+  },
+  "DomainPermissions": {
+    "mainText": "Allow {appName} to use your camera and microphone",
+    "secondaryText": "This is so participants can see and hear you.",
+    "linkText": "Need help? Get troubleshooting help"
```

Comment:
Design feedback - seems weird to say "help" twice here (and do regular users know what troubleshooting means? seems like quite a technical term), perhaps just "More help"